dvc/repo/init.py

```diff
@@ -43,7 +44,29 @@ def _welcome_message():
     logger.info(msg)
 
 
-def init(root_dir=os.curdir, no_scm=False, force=False, subdir=False):
+def generate_dvcignore(templates_list):
+    with open(".dvcignore", "w") as f:
+        f.write(
+            "# This is .dvcignore file."
+            " See more https://dvc.org/doc/user-guide/dvcignore\n"
+        )
```

Comment:
Good idea to create a default dvcignore. It'll make people aware of it. :+1: to that.

Regarding the comment, better to explain what it does. `This is .dvcignore file.` is clear from the filename.

Something like following, perhaps?
```suggestion
        f.write(
            "# add patterns of files dvc should ignore, can improve performance\n"
            "# Learn more at https://dvc.org/doc/user-guide/dvcignore\n"
        )
```

Reply:
@skshetry Please see my comment above. Maybe we should just use .gitignore directly. Would love to know what you think about that.